src/index.js

```diff
@@ -1,7 +1,8 @@
 // import _ from 'lodash';
 /* eslint-disable no-unused-vars */
```

Comment:
- [ ] Please pay careful attention to the feedback given by reviewers as I noticed the `eslint-disable no-unused-vars` hasn't still been removed please make sure to remove this as mentioned previously.